bin/picca_export.py

```diff
@@ -185,6 +185,7 @@ def main(cmdargs):
             z_dmat = z.copy()
         hdul.close()
     else:
+        dmat_name = 'DM'
```

Comment:
Why is this change here? Has this leaked from your separate work on the blinding strategy?

Reply:
The version of master I started working from had a bug related to blinding (now fixed I think) and this is me doing a temporary fix so this branch works. Before merging I'll pull master into this and get rid of this line.

Reply:
Sounds good.